### PR TITLE
getInternships/getting data from back-end

### PR DESCRIPTION
**Pull request by beliavskaja**
- added first data from back-end
- changed some texts in messages.json
- added an authorization header to all requests by default
- fixed comments

Screenshot 2021-11-08 at 19 23 57

### CODE REVIEW

File: src/store/reducers/internships.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { GET_INTERNSHIPS } from '../actions';
+
+const initialState = {
+  internships: [],
+  isLoading: false,
+};
+
+export default function internshipsReducer(state = initialState, action) {
```
Comment (svtslv17) [inline src/store/reducers/internships.js:8]: the reducer should be named as the state field that it changes

File: src/components/internshipCard/InternshipCard.jsx [hunk 2]
```diff
@@ -7,16 +7,22 @@ import {
...
   Typography,
 } from '@mui/material';
-import './InternshipCard.module.sass';
+import './InternshipCard.sass';
 import assets from '../../assets';
 import { getFieldLabel } from '../../utils';
 
+const dayjs = require('dayjs');
+
 export const InternshipCard = ({
```
Comment (svtslv17) [inline src/components/internshipCard/InternshipCard.jsx:16]: this "export" isn't needed here. You call it from below

File: src/components/internshipCard/InternshipCard.jsx [hunk 3]
```diff
@@ -25,17 +31,21 @@ export const InternshipCard = ({
...
       <Typography variant="subtitle2" color="text">
-        {getFieldLabel(numberOfMembers)}
+        {getFieldLabel('internships.program.selection')}
+        {maxCandidateCount}
+        {getFieldLabel('internships.program.members')}
       </Typography>
     </CardContent>
     <Box display="flex" flexDirection="row" justifyContent="center">
-      <Button className="internshipCardButton" size="small">
+      <Button raised className="internshipCardButton" size="small">
```
Comment (svtslv17) [inline src/components/internshipCard/InternshipCard.jsx:48]: are you sure the button needs this prop? 
image

Comment (beliavskaja) [reply src/components/internshipCard/InternshipCard.jsx:48]: you are right, no need, forgot to delete after changes in sass file

**Review by nsukhotskaya — APPROVED** [review]

**Review by ValeryiaMIRON — APPROVED** [review]

**Pull request merged**